core/src/apps/cardano/sign_tx.py

```diff
@@ -339,15 +374,15 @@ async def _process_tokens(
         seen_asset_name_bytes.add(token.asset_name_bytes)
         if should_show_tokens:
             await confirm_sending_token(ctx, policy_id, token)
-        builder.add_token(token.asset_name_bytes, token.amount)
+        hash_builder.add_item((token.asset_name_bytes, token.amount))
 
-    builder.finish_tokens()
+    hash_builder.finish_current_lazy_collection()
 
 
 async def _process_certificates(
     ctx: wire.Context,
     keychain: seed.Keychain,
-    builder: TxBuilder,
+    hash_builder: CborHashBuilder,
```

Comment:
I'd clarify at least by variable name that this is supposed to be "tx_hash_builder"